src/Common/src/Internal/Runtime/ModuleHeaders.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Internal.Runtime
+{
+    //
+    // ModuleHeaderSection IDs are used by the runtime to look up specific global data sections
+    // from each module linked into the final binary. New sections should be added at the bottom
+    // of the enum and deprecated sections should not be removed to preserve ID stability.
+    //
+    enum ModuleHeaderSection
+    {
+        StringEETypePtr             = 0x00,
```

Comment:
Could you please delete this, and replace the one place where it is used with `EETypeOf <string>` that Michal added recently?